exercises/concept/regular-chatbot/.meta/exemplar.js

```diff
@@ -0,0 +1,65 @@
+// @ts-check
+
+/**
+ * Given a certain command, help the chatbot recognize whether the command is valid or not.
+ *
+ * @param {string} command
+ * @returns {boolean} whether or not is the command valid
+ */
+
+export function isValidCommand(command) {
+  const regex = /^chatbot/i;
+  return regex.test(command);
+}
+
+/**
+ * Given a certain message, help the chatbot get rid of all the emoji's encryption throught the message.
+ *
+ * @param {string} message
+ * @returns {string} The message without the emojis encryption
+ */
+export function removeEmoji(message) {
+  const regex = new RegExp('emoji[0-9]+');
+  return message.split(regex).join('');
+}
+
+/**
+ * Given a certain phone number, help the chatbot recognize whether it is in the correct format.
+ *
+ * @param {string} number
+ * @returns {string} the Chatbot response to the phone Validation
+ */
+export function checkPhoneNumber(number) {
+  const regex =
+    /^\(\+?([0-9]{2})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{3})[-. ]?([0-9]{3})$/;
+  if (regex.test(number)) {
+    return 'Thanks! You can download me now on your Phone.';
+  } else {
+    return `Oops, it seems like I can't reach out to ${number}`;
+  }
+}
+
+/**
+ * Given a certain response from the user, help the chatbot get only the URL
+ *
+ * @param {string} userInput
+ * @returns {Array} all the possible URL's that the user may have answered
```

Comment:
```suggestion
 * @returns {string[]} all the possible URL's that the user may have answered
```

Reply:
Using the `string[]` value means that the returning value will be an Array which elements will be strings right? 

I also find it better and more secure this way instead of Array. Thanks for the suggestion.

Reply:
>Using the string[] value means that the returning value will be an Array which elements will be strings right?

Yes, exactly.